### PR TITLE
:bug: Removed indentation inside element

### LINKED ISSUE #31: :bug: indentation problem in XML files
**🐛 indentation problem in XML files**
some Elements in a xml file should not indent their inside component

**👀 To Reproduce**
1.  create a a xml, md or json file containing a codeblock
2. add every possible component inside it
3. generate a xml file

**🤷 Expected behavior**
All component inside the codeblock should be on the same line

**📸 Screenshots**
Actual behavior:
image
Expected behavior:
image ...

### PR DESCRIPTION
**Pull request by landryarki**
remove indentation of: codeblock, paragraph, code, italic, bold, link, image
see:
- #31

### CODE REVIEW

File: src/XML/WriterXML.hs [hunk 1]
```diff
@@ -60,22 +60,23 @@ elementsToXML [] _ = ""
 elementsToXML (x:xs) n = elementToXML x n ++ elementsToXML xs n
 
 elementToXML :: Element -> Int -> String
+elementToXML e (-1) = elementToXML e (-2)
 elementToXML (Text str) _ = str
-elementToXML (Bold elem) n = "<bold>" ++ elementToXML elem (n + 1) ++ "</bold>"
-elementToXML (Italic elem) n = "<italic>" ++ elementToXML elem (n + 1)
+elementToXML (Bold elem) n = "<bold>" ++ elementToXML elem (-1) ++ "</bold>"
```
Comment (axel-eck) [inline src/XML/WriterXML.hs]: Bold can contain other elements, not only text, is this not gonna reset the indentation ?

Comment (axel-eck) [reply src/XML/WriterXML.hs]: If this is the intended code, can you replace n by _ since you don't seem to use it

**Review by axel-eck — CHANGES_REQUESTED** [review]
Have you tested this? The behavior I can see coming out of this, is a bit strange.

**Review by 6im0n — APPROVED** [review]
Good for me :heavy_check_mark:

**Comment by landryarki** [comment]
> Have you tested this? The behavior I can see coming out of this, is a bit strange.

yes.

**Review by axel-eck — APPROVED** [review]

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Pull request merged**